api/tests/Serializer/PreventAutomaticEmbeddingPropertyMetadataFactoryTest.php

```diff
@@ -2,12 +2,13 @@
 
 namespace App\Tests\Serializer;
 
-use ApiPlatform\Core\Metadata\Property\Factory\PropertyMetadataFactoryInterface;
-use ApiPlatform\Core\Metadata\Property\PropertyMetadata;
-use ApiPlatform\Core\Metadata\Property\SubresourceMetadata;
+// use ApiPlatform\Metadata\Property\Factory\PropertyMetadataFactoryInterface;
+// use ApiPlatform\Metadata\Property\PropertyMetadata;
+// use ApiPlatform\Metadata\Property\SubresourceMetadata;
```

Comment:
i would remvoe these lines

Reply:
Fixed in https://github.com/ecamp/ecamp3/pull/3176/commits/7f42c2c199666c77d5f0c564570ab7aeef04d28e